packages/filter/src/components/tag/TagCondition.tsx

```diff
@@ -76,13 +76,15 @@ export const TagCondition: React.FC<ITagConditionProps> = ({
   }, [guid, onRemoveFilter]);
 
   return (
-    <TagConditionsWrapper>
-      <RemoveConditionButton onClick={onRemoveCondition}>
+    <TagConditionsWrapper className="at-filter--tag-conditions-wrapper">
```

Comment:
Is it internal component? I think you need pass classname prefix in classname props and use it in classnames. and i think the classname won't be uniq on the page, can you check it?